src/pages/singleDirection.tsx

```diff
@@ -3,26 +3,31 @@ import { NextPageContext } from 'next';
 import { parseCookies } from 'nookies';
 import Layout from '../layout/Layout';
 import { OPERATOR_COOKIE, SERVICE_COOKIE, JOURNEY_COOKIE, FARETYPE_COOKIE, PASSENGER_TYPE_COOKIE } from '../constants';
-import { deleteCookieOnServerSide } from '../utils';
 import { getServiceByNocCodeAndLineName, Service, RawService } from '../data/auroradb';
 import DirectionDropdown from '../components/DirectionDropdown';
 import { enrichJourneyPatternsWithNaptanInfo } from '../utils/dataTransform';
+import { ErrorInfo } from '../types';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
 
 const title = 'Single Direction - Fares Data Build Tool';
 const description = 'Single Direction selection page of the Fares Data Build Tool';
+const errorId = 'direction-error';
 
 interface DirectionProps {
     operator: string;
     passengerType: string;
     lineName: string;
     service: Service;
+    error: ErrorInfo[];
 }
 
-const SingleDirection = ({ operator, passengerType, lineName, service }: DirectionProps): ReactElement => {
+const SingleDirection = ({ operator, passengerType, lineName, service, error }: DirectionProps): ReactElement => {
     return (
         <Layout title={title} description={description}>
             <main className="govuk-main-wrapper app-main-class" id="main-content" role="main">
                 <form action="/api/singleDirection" method="post">
+                    <ErrorSummary errors={error} />
                     <div className="govuk-form-group">
```

Comment:
Same with this line being <div className={govuk-form-group ${error.length > 0 ? 'govuk-form-group--error' : ''}}> to get the red line alongside it